docs/source/conf.py

```diff
@@ -37,6 +37,11 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
 
+# copybutton options 
```

Comment:
Can you update the author and copyright?

Reply:
Done